gosafe.lic

```diff
@@ -0,0 +1,17 @@
+=begin
+=end
+
+custom_require.call(%w[common common-travel])
+
+class Gosafe
+  include DRC
+  include DRCT
+  include DRCI

```

Comment:
❌ Not in `custom_require` - it should be removed here or added there

Reply:
Removed includes

Reply:
Sorry I wasn't clear, I was only referring to `DRCI` (although `DRC` is also not necessary). I _think_ `include DRCT` is still necessary or I would expect to see an error related to Ruby not knowing which module `walk_to` is in. Can you test this for me?

Reply:
Build updated. it did need the DRCT. 

Oddly it was working without it before i closed and reopened the FE. is there a way to reload the lich without closing the restarting the FE